packages/components/addon/components/hds/app-footer/item.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<li class="hds-app-footer__item" ...attributes>
```

Comment:
thinking out loud: should `hds-app-footer__item` be `hds-app-footer__list-item` to make explicit that it's a child _item_ of the `hds-app-footer__list` element?